### PR TITLE
Added a PyLint Action

### LINKED ISSUE #263: Add code quality Actions
**Is your feature request related to a problem? Please describe.**
Code in the projects could be better,

**Describe the solution you'd like**
Add flake8 Github Action which will help new contributors to achieve better code.
Also, we could start gradually improving the quality of older projects.
The first step could be formatting with `black`.

### PR DESCRIPTION
**Pull request by vybhav72954**
# Description

As discussed in #263, I have added a PyLint quality check action. The `config` file in the `linters` folder can be edited to make it more apt for the Repo.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md` (The CONFIG file explains everything)
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works. (Check the Action Dashboard)

### CODE REVIEW

**Comment by vybhav72954** [comment]
@avinashkranjan and other mods, I allowed this action to run on the whole codebase this one time as proof that this action works properly. You can check the action dashboard for more info.

I like making such actions, and even incorporating multiple actions at once. I can make more GitHub actions if you want 😄

**Comment by vybhav72954** [comment]
I have modified the action so that it only runs on the new files in a Pull Request.

P.S. The stale action has now deprecated, shall I open an issue for that?

**Comment by atarax665** [comment]
> I have modified the action so that it only runs on the new files in a Pull Request.
> 
> P.S. The stale action has now deprecated, shall I open an issue for that?

Yeah sure, we would love to automate as many tasks as possible. That will ease the work of project maintainers and help to maintain consistency in the code base.

File: .github/linters/.python-lint.yml [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+
+# List of valid names for the first argument in a metaclass class method.
+valid-metaclass-classmethod-first-arg=mcs
+
+
+[EXCEPTIONS]
+
+# Exceptions that will emit a warning when being caught. Defaults to
+# "Exception"
+overgeneral-exceptions=Exception
```
Comment (atarax665) [inline .github/linters/.python-lint.yml:273]: Add a blank line at the EOF.

File: .github/workflows/py_lint.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      - name: Lint Code Base
+        uses: github/super-linter@v3
+        env:
+          VALIDATE_ALL_CODEBASE: false
+          VALIDATE_PYTHON: true
+          VALIDATE_PYTHON_PYLINT: true
+          DEFAULT_BRANCH: master
+          LINTER_RULES_PATH: /.github/linters
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PYTHON_PYLINT_CONFIG_FILE: .python-lint
```
Comment (atarax665) [inline .github/workflows/py_lint.yml:56]: Add a blank line at the EOF.

**Review by atarax665 — APPROVED** [review]
Just a minor suggestion everything else looks fine to me. :+1:

**Review by avinashkranjan — APPROVED** [review]
LGTM..🚀

**Comment by avinashkranjan** [comment]
> @avinashkranjan and other mods, I allowed this action to run on the whole codebase this one time as proof that this action works properly. You can check the action dashboard for more info.
> 
> I like making such actions, and even incorporating multiple actions at once. I can make more GitHub actions if you want 😄

Sure, If you see something which can be automated. Go ahead..It would help me a lot

**Pull request merged**